control_panel_api/management/commands/import_k8s_users.py

```diff
@@ -1,42 +1,76 @@
 import base64
 import logging
 
+from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.db.utils import IntegrityError
-from kubernetes import config, client
+from kubernetes import client, config
 
 from control_panel_api.models import User
+from moj_analytics.auth0_client import Auth0, ManagementAPI, User as Auth0User
 
 logger = logging.getLogger(__name__)
 
 
+def get_auth0_users():
+    api = Auth0(
+        settings.OIDC_CLIENT_ID,
+        settings.OIDC_CLIENT_SECRET,
+        settings.OIDC_DOMAIN
+    )
+    api.access(ManagementAPI(settings.OIDC_DOMAIN))
+
+    results = api.management.get_all(Auth0User, 'connection=github')
+
+    return results
```

Comment:
No need for a variable, just inline `return api.manage.get_all(...`

Reply:
Yeh, I was using it for debug purposes.